packages/odyssey-react/src/components/Button/Button.tsx

```diff
@@ -49,7 +49,8 @@ interface CommonProps
     | "danger"
     | "dismiss"
     | "dismissInverted"
-    | "clear";
+    | "clear"
+    | "affix";
```

Comment:
could we get a new simple storybook example for this variant? That helps us with VRT - Thank you!

Reply:
👍🏻